.github/workflows/test_definitions.yml

```diff
@@ -0,0 +1,27 @@
+name: Tests
```

Comment:
This file should be renamed to `tests.yml`

Reply:
Swapped with the other .yml file (refer to below comment), resolved